packages/wonder-blocks-tooltip/components/tooltip-portal-mounter.js

```diff
@@ -111,24 +102,31 @@ export default class TooltipPortalMounter extends React.Component<Props> {
         destination.parentNode.removeChild(destination);
     }
 
-    _timeoutDoMount() {
-        this._clearMountTimeout();
-        this._timeoutId = setTimeout(() => this._doMount(), 0);
-    }
+    _renderChildren() {
+        if (!this._destination) {
+            this._doMount();
+            return;
```

Comment:
If we have nowhere to mount, abandon this render and mount our portal. It will call us to render again once it has.

Reply:
Why not call `_doMount` from `componentDidMount`?

Reply:
I see... we may need to call `_doMount` more than once b/c `children` can change multiple times over the lifetime of this portal.  Could you add some documentation at the time that describes the life time of this component and when things are mounted/unmounted?

Reply:
It's not just that. We won't have an anchor element in the mount. That only appears after rendering.

Reply:
I realise that comment was confusing since render happens before mount.

Because the portal itself and its content are shown during a hover or focus, this generally happens after componentDidMount. However, there is a chance we might have content immediately, so I've reinstated a call in the componentDidMount and commented accordingly.

I think the renaming of methods and commentary will make things clearer.